### DISCUSSION TITLE
async/await

### DISCUSSION THREAD

**Discussion by darkartswizard** [webdriverio/webdriverio#7494] [Q&A]
Has anyone found away to bury async / await into a reusable wrapper library without having to touch every it(), describe() and page object function call?

**Comment by christian-bromann** [comment]
@darkartswizard I don't think this is possible. Every line where you use a WebdriverIO command you need to work with `async/await`. Given that we [can't support fibers](https://webdriver.io/blog/2021/07/28/sync-api-deprecation) anymore, this is the new reality unless Node.js decides to add such a feature again in on of their upcoming versions.

An option is to do similar things to Cypress where commands are queued in the order of their execution and then executed in the right order but this will start causing even more confusion because you won't be able to work with diffs and are sometimes required to nest calls.
